### ISSUE TITLE
5/5 HF1 - Hard CTD

### ISSUE THREAD

**Issue by Stalker012345** [LoneGazebo/Community-Patch-DLL#7835]
_1. Mod version (i.e Date - 4/14):_
5/5 HF1

_2. Mod list (if using Vox Populi only, leave blank):_
![image](https://user-images.githubusercontent.com/20593976/117548124-cbbf6380-aff8-11eb-8b15-95c10e2cfe75.png)

_3. Error description:_ 
Getting a hard CTD I cannot get past even going back a few turns.

_4. Steps to reproduce (optional):_
[Washington_0278 AD-1816.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6446269/Washington_0278.AD-1816.zip)

[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6446270/Logs.zip)

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. For instructions, go to the repository's main page, under "To enable logging for bug reports"):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves; you can change autosave frequency in the game's Options menu):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
just ran this to turn 281. When did it crash?

**Comment by Stalker012345** [comment]
On the turn I gave you....so that's weird

**Comment by LoneGazebo** [comment]
[CvGameCore_Expansion2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6452252/CvGameCore_Expansion2.zip)

This is a debug DLL, so it will be slow for gaming purposes, but try this and see if it crashes. If not, I'll send you a release DLL.

**Comment by Stalker012345** [comment]
New hard CTD on Turn 373 (still using your adjusted DLL). I thought it might be teh completion of hubble but even changing that timing doesn't fix it.

[auto.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6456620/auto.zip)
[Logs.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6456621/Logs.zip)

**Comment by LoneGazebo** [comment]
[CvGameCore_Expansion2.zip](https://github.com/LoneGazebo/Community-Patch-DLL/files/6463185/CvGameCore_Expansion2.zip)

**Comment by LoneGazebo** [comment]
TL;DR; firaxis method for getting spies is HORRIBLY unsafe. Direct reference to arrays that constantly change? No no, bad dog. Anyways, that was the problem, so I finally just up and fixed it.

**Comment by LoneGazebo** [comment]
@Stalker012345 did you test this?

**Comment by LoneGazebo** [comment]
@Stalker012345 ping ping
